scenario/src/main/kotlin/com/cosmotech/scenario/azure/ScenarioServiceImpl.kt

```diff
@@ -338,18 +338,18 @@ internal class ScenarioServiceImpl(
       addState: Boolean
   ): List<Scenario> {
     val currentUser = getCurrentAuthenticatedMail(this.csmPlatformProperties)
+    var templateQuery  = "SELECT * FROM c " +
+              "WHERE c.type = 'Scenario' AND c.workspaceId = @WORKSPACE_ID"
+    val params = mutableListOf(SqlParameter("@WORKSPACE_ID", workspaceId))
+    if (this.csmPlatformProperties.rbac.enabled) {
+      templateQuery  += " AND ARRAY_CONTAINS(c.security.accessControlList, {id: @ACL_USER}, true) " +
+             "OR c.security.default NOT LIKE 'none'"
+      params.add(SqlParameter("@ACL_USER", currentUser))
+    }
```

Comment:
Admin profile should be consider here :thinking: 
if RBAC is enable and the user is and admin (with no rights defined in ACL) he will not see the good scenario list